lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -681,9 +684,37 @@ def _push_alerts_to_relation_databag(self, relation: Relation) -> None:
 
         alert_rules_as_dict = alert_rules.as_dict()
 
+        alert_rules_as_dict = self._parse_and_add_extra_alerts(alert_rules_as_dict)
+
         if alert_rules_as_dict:
             relation.data[self._charm.app]["alert_rules"] = json.dumps(alert_rules_as_dict)
 
+    def _parse_and_add_extra_alerts(self, alert_rules_as_dict: dict) -> dict:
+        """Parse alerts from `self.extra_alerts_callable`.
+
+        Call `self.extra_alerts_callable`, parse alert groups and
+        extend (mutate) `alert_rules_as_dict` with new alerts.
+
+        Returns:
+            (dict) Modified `alert_rules_as_dict`. Return is done only for explicit mutation.
```

Comment:
```suggestion
            The original `alert_rules_as_dict`, augmented with the return value of `self.extra_alerts_callable`.
```

Reply:
"Augmented with" is very tortured for non-native speakers compared to "with the return value of `PrometheusRemoteWriteConsumer.extra_alerts_callable` merged into the dict" or similar. 

"Augmented with" (even as a native speaker) seems to imply that the actual contents of the existing `alert_rules_as_dict` somehow may have extra fields added (ala label injection), which this is not doing.

Reply:
fixed including suggestion from Ryan